### DISCUSSION TITLE
Having Long name and short name on problem sets

### DISCUSSION THREAD

**Discussion by pstaabp** [openwebwork/webwork3#3] [Ideas]
This was from Tani: I would like to suggest that an assignment have both a short "code name" (used in the URL, and elsewhere by the code) and a "long name" which gets displayed in the table of assignments, etc. The "long name" could include foreign characters and in general be more mean meaningful to a human without requiring the computer code to use it "everywhere".

**Comment by dlglin** [comment]
I'm not sure what benefit the short name provides.  We are already planning on having a unique ID for each set which is separate from the name.  I assumed that if a set identifier was included in the URL that it would be the set ID.
